styles/_components.css

```diff
@@ -94,3 +90,70 @@
     color: inherit;
     padding: 1rem;
 }
+
+.logo-header {
+    width: 4rem;
+    height: 4rem;
+    border-radius: 0.5rem;
+}
+
+.component {
+    font-weight: 300;
+}
+
+.big-heading {
+    font-size: 2.8rem;
+    letter-spacing: 0.06em;
+    font-weight: 600;
+    color: var(--color-primary-dark);
+}
+
+.normal-heading {
+    font-size: 2.2rem;
+    letter-spacing: 0.06em;
+    margin: .5rem 0;
+    color: var(--color-primary-dark);
+}
+
+.small-heading {
+    font-size: 1.8rem;
+    letter-spacing: 0.06em;
+    margin: .5rem 0;
+    color: var(--color-primary-dark);
+}
+
+.demo-box {
+    width: 100%;
+    padding: 1rem;
+    border: 1px solid var(--color-primary-dark);
+    border-radius: 1rem;
+}
+
+.code-container {
+    width: 100%;
+    min-height: 5rem;
+    background-color: var(--color-primary-dark);
+}
+
+.code-snippet-container {
+    width: 100%;
+    min-height: 10rem;
+    max-height: 25rem;
+    padding: 1rem 1rem 0 1rem;
+    background-color: #2b3e50;
+    overflow: auto;
+    border-radius: 1rem;
+}
+
+.iframe {
+    width: 100%;
+    height: auto;
+    overflow: hidden;
+    /* border: 1px solid var(--color-background); */
```

Comment:
remove comment if not intended for future reference

Reply:
sure in final merge I'll remove the whole comments